### ISSUE TITLE
Training from scratch 

### ISSUE THREAD

**Issue by ghost** [KaleidoZhouYN/Angular-Triplet-Loss#5]
Why do think we should do the training using pre-trained model not a new training from scratch?! 
"method 1:step
step 1:
train a pre-trained model use sphereface"

**Comment by KaleidoZhouYN** [comment]
It's hard to train triplet loss directly,so do angular triplet,always need lots of iteration to reach the final converge status.
while finetune on sphereface model won't have this problem.

**Comment by ghost** [comment]
I think this will decrease the ability of the generalization as you know, how you can solve this?

**Comment by KaleidoZhouYN** [comment]
yes，but the result show that sphereface constrain is too strong
